opencog/nlp/chatbot/sentence-matching.scm

```diff
@@ -0,0 +1,32 @@
+; Returns one or more sentences that are structurally similar to the input one.
```

Comment:
Two issues:
1) this is missing the usual license boilerplate
2) this is the wrong directory for this code. That chatbot directory contains an IRC chatbot interface, and nothing more.   Your code has nothing to do with IRC, and so doesn't belong.